pkg/analyzer/analyzers/github/github_test.go

```diff
@@ -36,82 +36,6 @@ func TestAnalyzer_Analyze(t *testing.T) {
 			want: `{
               "AnalyzerType": 0,
               "Bindings": [
-                {
```

Comment:
I confirmed that these entries were duplicates (at least comparing the `FullyQualifiedName` + `Permission` combo)